src/lambda/__tests__/fixtures/Lambda/LambdaFunctionThatReturnsJSONObject-fixture.js

```diff
@@ -16,6 +16,7 @@ export default class LambdaFunctionThatReturnsJSONObject {
     },
     service: {
       provider: {
+        arquitecture: "arm64",
```

Comment:
should this be `architecture` (spelling)?

Reply:
your wright, its `architecture`, i wrote it wrong.